src/components/graph-view.js

```diff
@@ -416,18 +416,21 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
         this.removeEdgeElement(edge);
       });
 
-      // remove node
+      // remove node - timeout helps for removing async rendered nodes
```

Comment:
https://github.com/trifacta/react-digraph/commit/5bdeac71f6560ceeb81b97d9f2cfa13ca7faff8a

Adding this for my reference...